guide/onboarding/funding-a-wallet.md

```diff
@@ -22,101 +30,300 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
+   alt-text = "Screen showing payment request."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/onboarding/funding-a-wallet/auto-setting.png"
+   retina = "/assets/images/guide/onboarding/funding-a-wallet/auto-setting@2x.png"
+   layout = "shadow"
+   caption = "The user has disabled automatic channel opens."
+   alt-text = "Screen showing disabled automatic channel opens toggle."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/onboarding/funding-a-wallet/manual-channel.png"
+   retina = "/assets/images/guide/onboarding/funding-a-wallet/manual-channel@2x.png"
+   layout = "shadow"
+   caption = "Once the on-chain funding payment is confirmed they can then manually open a channel with these funds."
+   alt-text = "Screen showing option to manually open a channel."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/onboarding/funding-a-wallet/find-peers.png"
+   retina = "/assets/images/guide/onboarding/funding-a-wallet/find-peers@2x.png"
+   layout = "shadow"
+   caption = "When manually opening a channel your application should help users find reliable peers."
+   alt-text = "Screen showing places to find Lightning peers."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/onboarding/funding-a-wallet/peer.png"
+   retina = "/assets/images/guide/onboarding/funding-a-wallet/peer@2x.png"
+   layout = "shadow"
+   caption = "A peer found by visiting amboss.com. The user copys the peers address to open a channel with them."
+   alt-text = "Screen showing a peer on amboss.com."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/onboarding/funding-a-wallet/open-channel.png"
+   retina = "/assets/images/guide/onboarding/funding-a-wallet/open-channel@2x.png"
+   layout = "shadow"
+   caption = "The user opens a channel with their selected peer."
+   alt-text = "Screen showing manual channel open UI."
+   width = 250
+   height = 541
+%}
 
-Users funding new wallets from an exchange could potentially come into contact with unknown concepts, such as a bitcoin address, QR code, or transaction confirmations. This is an excellent opportunity to explain the exchange to personal wallet flow.
+{% include picture.html
+   image = "/assets/images/guide/onboarding/funding-a-wallet/wallet-funded.png"
+   retina = "/assets/images/guide/onboarding/funding-a-wallet/wallet-funded@2x.png"
+   layout = "shadow"
+   caption = "Once a channel has been opened the wallet is ready to use. "
+   alt-text = "Screen showing that a payment channel has been opened."
+   width = 250
+   height = 541
+%}
 
-{% include /tip/close.html %}
+</div>
 
-{% include fact/pros.html %}
-  - An easy and popular way to acquire bitcoin
-{% include fact/close.html %}
+## Overview
```

Comment:
@Bosch-0 Yeah, I agree. I think if you put the entire flow at the top of the page, it would make more sense. Just showing the reader the entire flow before discussing it point-by-point would be helpful.

For example, per the [discussion in this thread](https://github.com/BitcoinDesign/Guide/pull/590/files#r768507191), the "How it works" section might make more sense if the reader has already seen an overview of the whole flow.